### ISSUE TITLE
Buttons: 텍스트 버튼 vs Link

### ISSUE THREAD

**Issue by dusskapark** [bridgedxyz/ui-dataset#7]
<img width="400" alt="Screen Shot 2020-11-17 at 2 53 59 PM" src="https://user-images.githubusercontent.com/4177529/99634980-afe4f700-2a84-11eb-8160-3f76a4e973e1.png">

링크`<a/>` 와 텍스트 only `<button>`은 한끗차이라서 맥락으로 이것이 버튼인지 아니면 링크인지를 구분해야 할 것 같습니다.
제 의견은 텍스트 only 버튼은 기본적으로는 라벨링하지 않고, 일부 예외를 가이드에 넣어두면 어떨까 합니다.

**Comment by dusskapark** [comment]
https://docs.google.com/document/d/1KSgi0wd4W3XgbqU1N88GOBaY4Ie0r2uEDttL72Fyg_w/edit?disco=AAAAHSw__ZY

**Comment by dusskapark** [comment]
<img width="428" alt="Screen Shot 2020-11-17 at 2 55 16 PM" src="https://user-images.githubusercontent.com/4177529/99635939-0f8fd200-2a86-11eb-948c-661c385e8d12.png">

이런 경우는 어떻게 해아할까요...ㅠ

**Comment by dusskapark** [comment]
<img width="472" alt="Screen Shot 2020-11-17 at 12 31 47 PM" src="https://user-images.githubusercontent.com/4177529/99636663-123ef700-2a87-11eb-8675-52615080b21e.png">

이메일이나 비번 찾기 기능도 통상적으로 쓰는 것 같습니다.

**Comment by dusskapark** [comment]
<img width="441" alt="Screen Shot 2020-11-17 at 12 49 04 PM" src="https://user-images.githubusercontent.com/4177529/99636743-31d61f80-2a87-11eb-9869-a7865afbe128.png">

위의 2개의 Primary / Secondary 버튼은 누가봐도 버튼인데..
그 아래에 Retrieve 버튼은 텍스트 버튼일까요.. 아니면 링크라고 봐야할까요?

**Comment by softmarshmallow** [comment]
IMO, If the plain text, (without container), should be considered as a TextButton if it is isolated and considered as clickable.
I.e. clickable text with non-clickable text should be considered as simply a RichText.

But in other sight, the model's performance by labelling them as a button might be considered as a NOIISE for machine to interpret.

**Comment by JongheeWoo** [comment]
I agree with @softmarshmallow . The component should be considered as a ghost button. Link should be underlined and should not have its container.

**Comment by dusskapark** [comment]
저... 한국어로 쓰셔도 됩니다. Jovin님이 구글 번역기를 쓰는게 낫다 싶어서 회의때 말해놨으니 ㅎㅎㅎ
